task/build_cli.js

```diff
@@ -0,0 +1,62 @@
+const fs = require('fs-extra')
+const path = require('path')
+const yargs = require('yargs')
+const build = require('../lib/build')
+const mkdirp = require('mkdirp')
+const args = yargs
+  .usage('Usage: $0 [options]')
+  .options({
+    style_dir: {
+      describe: 'The folder with the style inside it',
+      type: 'string',
+      demandOption: true,
+      nargs: 1
+    }
+  })
+  .options({
+    conf: {
+      describe: 'The conf file with the urls in it',
+      type: 'string',
+      demandOption: true,
+      nargs: 1
+    }
+  })
+  .help('h')
+  .alias('h', 'help')
+  .argv
+
+const buildDir = path.join(args.style_dir, 'build')
+const stylePath = path.join(args.style_dir,'style.json')
+const confStr = fs.readFileSync(args.conf, 'utf8')
+const styleStr = fs.readFileSync(stylePath, 'utf8')
+const style = JSON.parse(styleStr)
+const jsonconf = JSON.parse(confStr)
+
+let options = {
+  styleDir : args.style_dir,
+  conf : jsonconf,
```

Comment:
`options.conf` is defined here, whereas `options.config` is expected in `build_cli`